js/services/DataActions.ts

```diff
@@ -1,7 +1,8 @@
 import ObjectData from './ObjectData';
```

Comment:
I have changed it so that a single function is the default export and the load and save code has been encapsulated into separate "private" functions. I did not want these as named exports since I did not want to delegate the switch statement to the OfflineCore module (it's fat enough).